kori/app/dao/organization.py

```diff
@@ -0,0 +1,50 @@
+from uuid import UUID
+
+from kori.app.core.config import Settings
+from kori.app.db.connection import DbConnector
+from kori.app.models.organization import Organization
+from kori.app.schemas.organization import OrganizationCreate, OrganizationSchema, OrganizationUpdate
+from kori.app.utils.dict_utils import remove_null_values
+
+settings = Settings()
+
+db_connector = DbConnector(settings.DATABASE_URI)
+
+
+def create(organization_create: OrganizationCreate) -> OrganizationSchema:
+    session = db_connector.get_session()
+    new_org_db = Organization(**organization_create.dict())
+
+    session.add(new_org_db)
+    session.commit()
+
+    return OrganizationSchema.from_orm(new_org_db)
+
+
+def get_organization_by_id(organization_id: UUID) -> OrganizationSchema | None:
+    session = db_connector.get_session()
+    organization = session.query(Organization).get(organization_id)
+    return OrganizationSchema.from_orm(organization)
```

Comment:
Case where no organization is not handled?

Or am I missing some behavior of Pydantiic's `from_orm` ?